FEATURE/TariikMVP/LoginResponse

TarikVelic opened the pull request:
Login Response for Mobile

--------------------------------------------------------------------
 MAIN
 * Added login response for mobiles
![Screenshot_17](https://user-images.githubusercontent.com/101573891/234217456-80af7e61-b808-4000-ad61-fc3698782bb1.png)

--------------------------------------------------------------------
 OTHER
 * 
--------------------------------------------------------------------

TarikVelic committed:
Login Response for Mobile

--------------------------------------------------------------------
 MAIN
 * Added login response for mobiles
--------------------------------------------------------------------
 OTHER
 *
--------------------------------------------------------------------

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+        height: 100px;
+      }
+      .login-right {
+        max-width: 400px;
```
Please use % instead of pixels

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+      .logotype-mobile {
+        margin-top: 50px;
+        margin-bottom: 20px; 
+        width: 300px;
```
please use % instead of pixels for width

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
     padding-bottom: 10px;
     }
 
+    @media only screen and (max-width: 767px) {
```
You can view this topic and work like this https://medium.com/codeartisan/breakpoints-and-media-queries-in-scss-46e8f551e2f2

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+        justify-content: center;
+      }
+      
+      .login-buttons button {
```
we use scss so you can write this like 
```
.login-buttons{
button{
}

}
```

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+      }
+      
+      .login-buttons button {
+        background: #1A3835;
```
check if this color already exist in file with colors and reuse it. if not then create new scss variable and use it here

becarselman replied:
You have example in this file in line 173

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+      .login-buttons button {
+        background: #1A3835;
+        border-radius: 6px;
+        color: #fff;
```
same

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+        text-align: right;
+        letter-spacing: -0.015em;
+        text-decoration-line: underline;
+        color: #1A3835;
```
same

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+        font-size: 14.5px;
+        line-height: 24px;
+        text-align: right;
+        letter-spacing: -0.015em;
```
don't use em, and try to avoid negative values. Also create scss mixin for fonts for example
mixin for bold font for medium and normal and then just include font that are you want so you don't need to write font-style, font-weight...

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+      .login-buttons a {
+        font-style: normal;
+        font-weight: 500;
+        font-size: 14.5px;
```
Why 14.5px. I think difference between 14 and 14.5 is to small.

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+      }
+      .forgot-password {
+        margin-left: auto;
+        margin-top: 1px;
```
margin top 1px? I think 1px is imperceptibly.

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+        font-size: 16px;
+      }
+        .remember-password {
+          font-size: 12.5px;
```
same as previous comment

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+      }
+        .remember-password {
+          font-size: 12.5px;
+          margin-left: -6px;
```
same as previous comment

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+      }
+      
+      #remember-password {
+        font-size: 12.5px;
```
same

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+          
+      }
+      
+      #remember-password {
```
can we use classNames instead of id

becarselman commented on src/components/loginform/LoginForm.scss:
```diff
@@ -186,4 +186,72 @@ opacity: 1;
+        display: flex;
+        align-items: center;
+      }
+      .login-buttons a {
```
same as previous comment

TarikVelic committed:
FEATURE/TariikMVP/LoginResponse

--------------------------------------------------------------------
 MAIN
 * Added mixin mobile for @media
   Added mixin for styles
   Removing unnecessery stuff from scss
--------------------------------------------------------------------
 OTHER
 *
--------------------------------------------------------------------

TarikVelic committed:
FEATURE/TariikMVP/LoginResponse

--------------------------------------------------------------------
 MAIN
 * Added custom mixins for media
--------------------------------------------------------------------
 OTHER
 * <other msg>
--------------------------------------------------------------------

TarikVelic committed:
FEATURE/TariikMVP/LoginResponse

--------------------------------------------------------------------
 MAIN
 * Small changes
--------------------------------------------------------------------
 OTHER
 * <other msg>
--------------------------------------------------------------------

TarikVelic committed:
FEATURE/TariikMVP/LoginResponse

--------------------------------------------------------------------
 MAIN
 * fix
--------------------------------------------------------------------
 OTHER
 * <other msg>
--------------------------------------------------------------------

TarikVelic committed:
FEATURE/TariikMVP/LoginResponse

--------------------------------------------------------------------
 MAIN
 * Changed font
   Changed width for inputs
--------------------------------------------------------------------
 OTHER
 * <other msg>
--------------------------------------------------------------------

ElvaRivle reviewed (approved):
Great work
